### PR TITLE
Ideas for Refactoring

### PR DESCRIPTION
**Pull request by Anselmoo**
Dear AutoKeras teams, this PR should provide ideas for code refactoring, which are bump in one single PR

### CODE REVIEW

File: autokeras/blocks/preprocessing.py [hunk 1]
```diff
@@ -211,9 +211,9 @@ def build(self, hp, inputs=None):
             flip_mode = ""
         elif horizontal_flip and vertical_flip:
             flip_mode = "horizontal_and_vertical"
-        elif horizontal_flip and not vertical_flip:
+        elif horizontal_flip:
             flip_mode = "horizontal"
-        elif not horizontal_flip and vertical_flip:
+        else:
```
Comment (haifeng-jin) [inline autokeras/blocks/preprocessing.py:216]: The redundancy here is for readability.
I prefer to leave it as it was.

**Review by haifeng-jin — CHANGES_REQUESTED** [review]
Thanks for the PR! After addressing the comments, the PR should be good to merge.

File: autokeras/blocks/preprocessing.py (hunk 1 above)
Comment (Anselmoo) [reply autokeras/blocks/preprocessing.py:216]: @haifeng-jin I understand that, then better close the PR

Comment (Anselmoo) [reply autokeras/blocks/preprocessing.py:216]: @haifeng-jin sorry misunderstanding, you mean the preprocessing should stay as it is and then you are fine?

**Review by haifeng-jin — APPROVED** [review]
LGTM

**Pull request merged**